lib/src/mount.rs

```diff
@@ -57,3 +57,19 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
         [dev, target.as_str()],
     )
 }
+
+// Ensure the `/dev` directory exists.
+fn ensure_mountpoint(target: &str) -> Result<()> {
+    std::fs::create_dir_all(target)?;
+    Ok(())
+}
+
+/// General mount that also checks that mount path exists
```

Comment:
```suggestion
/// Create the target directory if it doesn't exist, then mount the specified filesystem
```

(It doesn't "check", it "ensures" or so right?)